src/main/scala/com/metl/utils/FunctionConverter.scala

```diff
@@ -0,0 +1,9 @@
+package com.metl.utils
+
+object FunctionConverter {
```

Comment:
Nice.  Did you use it though?  Perhaps you'd also want to do the reverse implicit conversion too?